pkg/downloader/chart_downloader.go

```diff
@@ -97,6 +98,13 @@ func atomicWriteFile(filename string, body io.Reader, mode os.FileMode) error {
 	return os.Rename(tempName, filename)
 }
 
+type InMemoryChart struct {
```

Comment:
Can you move this to before the first function declaration, as well as the associated variable `InMemoryCache`.

Reply:
Done